pageserver/src/metrics.rs

```diff
@@ -287,6 +287,15 @@ pub static TENANT_TASK_EVENTS: Lazy<IntCounterVec> = Lazy::new(|| {
     .expect("Failed to register tenant_task_events metric")
 });
 
+pub static BROKEN_TENANT_COUNT: Lazy<IntGaugeVec> = Lazy::new(|| {
+    register_int_gauge_vec!(
+        "pageserver_broken_tenant_count",
+        "Number of broken tenants",
+        &["tenant_id"]
+    )
+    .expect("Failed to register broken_tenant_count metric")
+});
+
```

Comment:
Shouldn't this be just `IntGauge` rather than `IntGaugeVec`? We don't want a gauge per tenant, the gauge for one tenant would always be 1 or 0.

But come to think of it, it would be nice to have a gauge for the other states too, not just broken tenants. What do you think of this?
```suggestion
pub static BROKEN_TENANT_COUNT: Lazy<IntGaugeVec> = Lazy::new(|| {
    register_int_gauge_vec!(
        "pageserver_tenant_state_count",
        "Number of tenants per state",
        &["state"]
    )
    .expect("Failed to register tenant_state_count metric")
});

```